projects/docs/src/app/docs-wrapper/pages/tooltip-page/tooltip-page.component.html

```diff
@@ -47,5 +59,18 @@
                 <option *ngFor="let item of themeList" [value]="item.value">{{ item.label }}</option>
             </select>
         </label>
+        <label class="w-100">
+            Cascade
+            <p><small>Add mode flexible positions to the tooltip (top, top-right, bottom-right...), default is false</small></p>
+            <sebng-checkbox [(ngModel)]="cascade" label="Cascade"></sebng-checkbox>
+        </label>
+
+        <label class="w-100">
```

Comment:
no need to wrap these controls in a label just us the checkbox with the label and description directly

Reply:
> no need to wrap these controls in a label just us the checkbox with the label and description directly

Agree

Reply:
fixed